src/env.ts

```diff
@@ -4,22 +4,24 @@ export function colossusEndpoint() {
   return process.env.VTEX_COLOSSUS_ENDPOINT || `https://infra.io.vtex.com/colossus/v0`
 }
 
-export function region(): string {
-  return process.env.VTEX_CLUSTER || Region.Production
-}
-
 export function cluster() {
   return process.env.VTEX_CLUSTER || getCluster() || ''
 }
 
+export function region(): string {
+  return cluster() || Region.Production
```

Comment:
Sanity check: Is `''` treated as "falsy"?

Reply:
Yes:
```
node -e "console.log('' == false)"
```